Sources/GDSCommon/Patterns/GDSInformation/GDSInformationViewController.swift

```diff
@@ -21,29 +21,90 @@ public final class GDSInformationViewController: BaseViewController, TitledViewC
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+        
+    @IBOutlet private var bottomStack: UIStackView!
+    @IBOutlet private var scrollView: UIScrollView!
     
-    @IBOutlet private var informationImage: UIImageView! {
-        didSet {
-            let font = UIFont(style: .largeTitle, weight: viewModel.imageWeight ?? .semibold)
-            let configuration = UIImage.SymbolConfiguration(font: font, scale: .large)
-            informationImage.image = UIImage(systemName: viewModel.image, withConfiguration: configuration)
-            informationImage.tintColor = viewModel.imageColour ?? .gdsPrimary
-            informationImage.accessibilityIdentifier = "information-image"
+    private var hasFootnoteBeenRemoved = false
+    
+    public override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        
+        checkBottomStackHeight()
+    }
+    
+    private func checkBottomStackHeight() {
```

Comment:
I'm wondering if we should this or similar method on the `BaseViewController` so we can more easily implement this on other screens. I think at it's most basic, it could be a method that returns `true` or `false` for a given check.

Reply:
I've made a note about this in the deck for the Mobile Design System - also made a note about the patterns when landscape :)  Design use it as a reference when looking for details about the components patterns we have in Common